openlibrary/templates/type/author/edit.html

```diff
@@ -97,7 +97,7 @@ <h1>$_("Edit Author")</h1>
                         <div class="formElement">
                             <div class="label">
                                 <label for="alternate_names">$_("Does this author go by any other names?")</label>
-                                <span class="tip">$_("For example:") Lev Nikolayevich Tolstoy; Yasnaya Polyana</span>
+                                <span class="tip">$_("For example:") Lev Nikolayevich Tolstoy was also known as Leo Tolstoy. Please list one name per line.</span>
```

Comment:
```suggestion
                                <span class="tip">$_("For example: Lev Nikolayevich Tolstoy was also known as Leo Tolstoy. Please list one name per line.")</span>
```
Please put it in the $_ like this so it can be translated :)

Reply:
Thank you for your guidance and patience. I have made the suggested changes. Please review and let me know if any further adjustments are needed.